fiftyone/core/models.py

```diff
@@ -250,6 +250,7 @@ def apply_model(
                 skip_failures,
                 filename_maker,
                 progress,
+                field_mapping=kwargs.get("field_mapping", None),
```

Comment:
@jacobsela doesn't this code:

https://github.com/voxel51/fiftyone/blob/81c954b1292dd2e67d57b096cdd8287d67462eee/fiftyone/core/models.py#L198-L202

mean that, as currently written, `FiftyOneTorchDataset` will only ever get `samples.select_fields()` passed to it, and therefore that models that use `required_key` to specify non-default fields won't have access to those fields and thus inference won't work?